src/templates/registration.html

```diff
@@ -10,7 +10,7 @@ <h1>Register</h1>
         </p>
         <p>
             {{ form.password.label }}<br>
-            {{ form.password(size=32) }}
+            {{ form.password(size=64) }}
```

Comment:
If you put 64 here because of the database field size it has almost no sense, because database stores a _hash_, but user provides an actual _password_. `sha256` hash of anything will always be 64 chars long, no matter how long the value of source.

For example:
- hash of 5 chars string "hello"
  will be `2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824`
- hash of 100 chars string "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean eu ligula auctor, fringilla nisi eu."
  will be `8608f66e4eb9908a149d8b4beea5ef39b91f1849c52925d3d50fa1e4fea2a7cd`

So we don't actually need to force user to provide a password with length no more than 64. Password length can be anything. And let's make this field size like 200 chars, to provide a users an ability to create password which contain the whole phrases, and also to be sure that even 200 chars password fit in our database after hashing is applied.